src/api/CapsulaHubArgs.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * The arguments that are used for "run" and "build" command of the app
+ */
+import { ConfigurationType } from './ConfigurationType';
+
+export interface CapsulaHubArgs {
+  /**
+   * The path to the configuration (the format of the path depends on configurationType)
+   */
+  configurationPath: string;
```

Comment:
Not clear what is this argument `configurationPath`.

Reply:
configurationPath === token